src/lib/components/submit/SubmitAddNodeProvider.svelte

```diff
@@ -90,6 +91,61 @@
 
 			debounceSave();
 		})();
+
+	export function checkFields(): boolean {
```

Comment:
- for consistency reason, can you use an arrow function please?
- the return type is "boolean", what means "true" and what means "false"? 
- this can probably be finds place elsewhere than within a component, in a service maybe?